### DISCUSSION TITLE
optimal number of cluster for scATAC

### DISCUSSION THREAD

**Discussion by pengxin2019** [GreenleafLab/ArchR#1059] [Questions / Documentation]
Hi Ryan,
I am wondering whether you have any suggestions on the optimal number of clusters for scATAC. How could be get the optimal number of cluster for scATAC before its integration with scRNA?

Any statistics or packages are recommended?
I know that "clustree" has been developed for this.. but any other tool can give us more clear answer?

Thanks
Best
penny

**Comment by rcorces** [comment]
There is no optimal number - it really depends on your application. It can be helpful to assess your clusters and try to reclassify them based on known cell types or cell states. There are packages out there that try to identify the optimal number of clusters but I'm not aware of anything that would easily work with ArchR.
